ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/dynamic-prefix-lists.j2

```diff
@@ -0,0 +1,25 @@
+{% if dynamic_prefix_lists is arista.avd.defined %}
+
+## Dynamic Prefix-lists
+
+### Dynamic Prefix-lists Summary
+
+{%     for dynamic_prefix_list in dynamic_prefix_lists | arista.avd.natural_sort('name') %}
+#### {{ dynamic_prefix_list.name }}
+
+{%         if dynamic_prefix_list.prefix_list.ipv4 is arista.avd.defined %}
+{%             set prefix_list = dynamic_prefix_list.prefix_list.ipv4 %}
+{%         elif dynamic_prefix_list.prefix_list.ipv6 is arista.avd.defined %}
+{%             set prefix_list = dynamic_prefix_list.prefix_list.ipv6 %}
+{%         endif  %}
+| Match Map | Prefix-list |
```

Comment:
Please don't create a separate table for every entry in ** dynamic_prefix_lists**. Lets say we have 50 entries => 50 tables, which will overflow and time consuming.

Please use only one table with the first row being the **dynamic_prefix_list.name** instead of it being the heading.

Also use the if with a defined test after line 6 to make sure the table will be printed only if the dynamic_prefix_lists is defined.

Reply:
Also after the refactor please move the blank line6 below that if defined test.

Reply:
Good catch. Thanks